setup.cfg

```diff
@@ -4,7 +4,7 @@ paths = features/
 show_skipped = no
 
 [flake8]
-exclude = docs,.cache,.tox,*.egg-info
+exclude = docs,.cache,.tox,*.egg-info,.ropeproject
 
-[pytest]
+[tool:pytest]
```

Comment:
As soon as a new version of `behave` is released we could actually consider moving the whole content of [setup.cfg](https://github.com/behave/behave-django/blob/master/setup.cfg) to [tox.ini](https://github.com/behave/behave-django/blob/master/tox.ini).

All tools we use for testing also read their sections out of `tox.ini`, including `behave` in its [development version](https://github.com/behave/behave/pull/447).